tests/testthat/test-opart-gaussian.R

```diff
@@ -17,8 +17,8 @@ test_that("opart gives error when data points are less than 1", {
   fixed=TRUE)
 })
 
-#test for small penalty
-res <- opart_gaussian(c(1,2,3,4,5), 0.0001)
+#test for zero penalty
```

Comment:
@tdhock , made the penalty 0 for small penalty.

Reply:
why? what do you expect for penalty=0.0001?

Reply:
@tdhock , for 0.0001 we should still get all data points as segment ends. However, I made this to 0 to make sure we are using the smallest penalty value possible.